db/sql/cassandra/README.md

```diff
@@ -22,6 +22,18 @@ Also, the client can configure following timeouts:
 - RedialInterval
     - If not zero, gocql attempt to reconnect known DOWN nodes in every ReconnectSleep (ReconnectInterval)
     - Default value is 60s
+- Example
+```go
+    config := &cassandra.Config{
+        Endpoints:      endpoints,
+        Port:           port,
```

Comment:
We have agreed on that you will unify port to be consistent with etcd, redis...

Please refactor...

Reply:
Will refactor in a separate commit